format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSinkConfig.java

```diff
@@ -84,13 +84,32 @@ public void validate(FailureCollector collector) {
           .withConfigProperty(NAME_SUFFIX).withStacktrace(e.getStackTrace());
       }
     }
+
+    FileFormat format = FileFormat.CSV;
```

Comment:
Was the default CSV? Or are we changing it to CSV?

Reply:
We are not changing from what it is. But, in any case changed it to JSON (which is the default).